simple-service-webapp-tests/src/test/java/com/example/ConnectionUtils.java

```diff
@@ -0,0 +1,52 @@
+package com.example;
+
+
+import com.example.rest.authentication.Credentials;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class ConnectionUtils {
+
+    private String path;
+
+    public ConnectionUtils() {
+        this.path = "C:\\simple-service-webapp\\simple-service-webapp-tests\\src\\test\\resources\\connection.properties";;
```

Comment:
can this be a relative url? if I checkout the code on my machine and try to run it, it won't work because of this